packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -853,6 +857,20 @@ export default defineComponent({
 					return;
 				}
 
+				if (
+					parameterData.type &&
+					SHOULD_CLEAR_NODE_OUTPUTS[nodeType.name] &&
+					SHOULD_CLEAR_NODE_OUTPUTS[nodeType.name].eventTypes.includes(parameterData.type) &&
+					SHOULD_CLEAR_NODE_OUTPUTS[nodeType.name].parameterPaths.includes(parameterData.name)
+				) {
+					this.workflowsStore.clearNodeOutgoingConnections(node.name);
+					this.showMessage({
```

Comment:
IMO, we should also check if the node has any outgoing connections and not show this error every time, as it gets a bit annoying when moving for more than a single position:

https://github.com/n8n-io/n8n/assets/12657221/8e1578ff-9112-489b-88c0-664cd2ce4034